dkist/tests/test_benchmarks.py

```diff
@@ -35,3 +32,9 @@ def plot_and_save_fig(ds=visp_dataset_no_headers, axes=axes):
         ds.plot(plot_axes=axes)
         plt.savefig("tmpplot")
         plt.close()
+
+
+@pytest.mark.benchmark
+def test_dataset_compute_data(benchmark, real_visp):
+    ds = real_visp[0, :5]
```

Comment:
Given this benchmark now runs in under 1s maybe this is too little data to get through the overhead of the dask array and stuff? Maybe make it :15 or something?